crawl-ref/source/rltiles/dc-mon.txt

```diff
@@ -894,6 +894,7 @@ bush_burning MONS_BUSH_BURNING
 %rim 1
 oklob_sapling MONS_OKLOB_SAPLING
 oklob_plant MONS_OKLOB_PLANT
+starflower_02 MONS_STARFLOWER
```

Comment:
Why two different tiles? (Can't preview easily in the GitHub app, if you intend for them to animate this isn't how to do that.)

Reply:
oh, one was kindof a joke (demonic plant tile holding rift), the other is a recolored oklob, which is probably a better placeholder, but tastes vary so i just committed them both for now!